bazel/envoy_build_system.bzl

```diff
@@ -398,54 +399,8 @@ def _proto_header(proto_path):
     return None
 
 # Envoy proto targets should be specified with this function.
-def envoy_proto_library(
-        name,
-        srcs = [],
-        deps = [],
-        external_deps = [],
-        generate_python = True):
-    # Ideally this would be native.{proto_library, cc_proto_library}.
-    # Unfortunately, this doesn't work with http_api_protos due to the PGV
-    # requirement to also use them in the non-native protobuf.bzl
-    # cc_proto_library; you end up with the same file built twice. So, also
-    # using protobuf.bzl cc_proto_library here.
-    cc_proto_deps = []
-    py_proto_deps = ["@com_google_protobuf//:protobuf_python"]
-
-    if "api_httpbody_protos" in external_deps:
-        cc_proto_deps.append("@googleapis//:api_httpbody_protos")
-        py_proto_deps.append("@googleapis//:api_httpbody_protos_py")
-
-    if "http_api_protos" in external_deps:
-        cc_proto_deps.append("@googleapis//:http_api_protos")
-        py_proto_deps.append("@googleapis//:http_api_protos_py")
-
-    if "well_known_protos" in external_deps:
-        # WKT is already included for Python as part of standard deps above.
-        cc_proto_deps.append("@com_google_protobuf//:cc_wkt_protos")
-
-    cc_proto_library(
-        name = name,
-        srcs = srcs,
-        default_runtime = "@com_google_protobuf//:protobuf",
-        protoc = "@com_google_protobuf//:protoc",
-        deps = deps + cc_proto_deps,
-        # Avoid generating .so, we don't need it, can interfere with builds
-        # such as OSS-Fuzz.
-        linkstatic = 1,
-        alwayslink = 1,
-        visibility = ["//visibility:public"],
-    )
-
-    if generate_python:
-        py_proto_library(
-            name = name + "_py",
-            srcs = srcs,
-            default_runtime = "@com_google_protobuf//:protobuf_python",
-            protoc = "@com_google_protobuf//:protoc",
-            deps = deps + py_proto_deps,
-            visibility = ["//visibility:public"],
-        )
+def envoy_proto_library(name, **kwargs):
+    return api_proto_library(name, visibility = ["//visibility:public"], **kwargs)
```

Comment:
That doesn't quite work, because `api_proto_library` overrides defaults itself; untangling that ball of string is best left as an exercise for another PR :)